gammapy/modeling/models/temporal.py

```diff
@@ -472,79 +471,93 @@ def __str__(self):
         )
 
     @classmethod
-    def read(cls, path):
-        """Read lightcurve model table from FITS file.
+    def from_table(cls, table):
+        """Create a Template model from an astropy table"""
+        columns = [_.upper() for _ in table.colnames]
+        t_ref = table.meta["MJDREFI"]
```

Comment:
This only copies over the integer part of the reference time. Please rely on https://github.com/gammapy/gammapy/blob/master/gammapy/utils/time.py#L18 instead